db/seed.js

```diff
@@ -24,50 +27,56 @@ var mediaSchema = mongoose.Schema({
 var Item = mongoose.model('Item', mediaSchema);
 
 var insertVideos = (arr) => {
-  for (let i = 0; i < 20; i++) {
+  for (let i = 0; i < 200; i++) {
```

Comment:
A recommendation here would be to make 200 a `const` variable and initialize it at the top of the page. This practice would help if say you needed to update the total number of videos per game and if many other functions utilized the same value. It allow you to just modify that one value and ensure all the functions that reference the same value are updated.